[Bug] - post install script's mkdir should be recursive

When [installing the kernel]((scripts/install/kernel_spec.js#L34) in the post install script, the `fs.mkdirSync(kernelDir)` should be `fs.mkdirSync(kernelDir, { recursive: true })` for cases where the directory `~/Library/Jupyter/kernels` doesn't exist.

Example log:
```shellsession
$ npm i

> nelu-kernelu@5.1.0 postinstall /Users/josungbin/Documents/Workspace/nelu-kernelu
> node ./scripts/install

fs.js:130
    throw err;
    ^

Error: ENOENT: no such file or directory, mkdir '/Users/josungbin/Library/Jupyter/kernels/javascript'
    at Object.mkdirSync (fs.js:834:3)
    at KernelSpec.install (/Users/josungbin/Documents/Workspace/nelu-kernelu/scripts/install/kernel_spec.js:34:16)
    at Object.<anonymous> (/Users/josungbin/Documents/Workspace/nelu-kernelu/scripts/install/index.js:5:12)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10)
    at internal/main/run_main_module.js:17:11 {
  errno: -2,
  syscall: 'mkdir',
  code: 'ENOENT',
  path: '/Users/josungbin/Library/Jupyter/kernels/javascript'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nelu-kernelu@5.1.0 postinstall: `node ./scripts/install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nelu-kernelu@5.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/josungbin/.npm/_logs/2019-09-18T06_20_27_111Z-debug.log

$ mkdir ~/Library/Jupyter/kernels

$ npm i

> nelu-kernelu@5.1.0 postinstall /Users/josungbin/Documents/Workspace/nelu-kernelu
> node ./scripts/install

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN nelu-kernelu@5.1.0 license should be a valid SPDX license expression

audited 192 packages in 0.883s
found 0 vulnerabilities
```

--- a/scripts/install/kernel_spec.js
+++ b/scripts/install/kernel_spec.js
@@ -31,7 +31,7 @@ class KernelSpec {
         const { kernelDir, jsonPath, imgDir } = this._getIntsallPaths();
 
         if (!fs.existsSync(kernelDir)) {
-            fs.mkdirSync(kernelDir);
+            fs.mkdirSync(kernelDir, { recursive: true });
         }
 
         const kernelSpec = JSON.stringify({ argv, display_name, language });

